history.py

```diff
@@ -0,0 +1,26 @@
+from django.conf import settings
+from django_mako_plus import view_function, jscontext
+from catalog import models as cmod
+
+
+class LastFiveMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+        # One-time configuration and initialization.
+
+    def __call__(self, request):
+        # product_ids = []
+        product_ids = request.session.get('lastfivelist', [])
+        product_objs = []
+        for item in product_ids:
```

Comment:
Naming things "item" isn't super conventional. I suggest naming variables more specifically so your code is easier to read

Reply:
Thanks for the tip! I should work on that for sure.